tests/test_crypto.py

```diff
@@ -1935,13 +1935,10 @@ def test_digest(self):
         """
         cert = load_certificate(FILETYPE_PEM, old_root_cert_pem)
         assert (
-            # This is MD5 instead of GOOD_DIGEST because the digest algorithm
-            # actually matters to the assertion (ie, another arbitrary, good
-            # digest will not product the same digest).
             # Digest verified with the command:
-            # openssl x509 -in root_cert.pem -noout -fingerprint -md5
-            cert.digest("MD5")
-            == b"19:B3:05:26:2B:F8:F2:FF:0B:8F:21:07:A8:28:B8:75"
+            # openssl x509 -in root_cert.pem -noout -fingerprint -sha256
+            cert.digest("SHA256")
+            == b"3E:0F:16:39:6B:B1:3E:4F:08:85:C6:5F:10:0D:CB:2C:25:C2:91:4E:D0:4A:C2:29:06:BD:55:E3:A7:B3:B7:06"
```

Comment:
This will fail our linter if it exceeds 79 columns

Reply:
`black` may format it correctly, or failing that you can split a string literal over lines with parens + implicit line continuations, e.g.

```
x = (
    b"..."
    b"more content"
(
```

Reply:
Done. (`black` doesn't seem to do it automatically.)